instagram/instagram/src/Components/dataLayout.js

```diff
@@ -1,14 +1,31 @@
 import React from 'react';
+
 import PropTypes from 'prop-types';
 
 function PostContainer(props) {
     return (
         <>
+            <div className='instagram'>
+                <p><strong>Instagram</strong></p>
+            </div>
             {props.dummyData.map(dummyData => (
                 <div key={dummyData.id} className="posts">
-                    <img src={dummyData.thumbnailUrl} alt={dummyData.username} />
-                    <p>{dummyData.username}</p>
-                    <img src={dummyData.imageUrl} alt={dummyData.username} />
+                    {/* username and profile image */}
+
+                    <div className='top'>
+                        <img className='profile' src={dummyData.thumbnailUrl} alt={dummyData.username} />
+                        <p><strong>{dummyData.username}</strong></p></div>
+                    <img className='picture' src={dummyData.imageUrl} alt={dummyData.username} />
+
+                    {/* likes */}
+                    <p className='likes'>{dummyData.likes} Likes</p>
+
+                    {/* comments */}
+                    <div className='comments'>
+                        {dummyData.comments &&
```

Comment:
this condition here is missing the `.length`  should be `dummyData.comments.length` but I am going to assume that that first condition is never being met, or always evaluating to true. so it is always going to revert to the second condition whether or not the comments array is empty or not.